docs/standards/universal-profile/lsp0-erc725account.md

```diff
@@ -49,8 +49,8 @@ The following types of calls (= operation types) are available:
 
 | Operation number |                     Operation type                     | Description                                                                                                                             |
 | :--------------: | :----------------------------------------------------: | :-------------------------------------------------------------------------------------------------------------------------------------- |
-|        0         |                         `CALL`                         | call another smart contract                                                                                                             |
-|        1         |                        `CREATE`                        | create a new smart contract with the associated bytecode passed as `_data`                                                              |
+|        0         |          [`CALL`](https://www.ethervm.io/#F1)          | call another smart contract                                                                                                             |
```

Comment:
This is a really great idea and improvement!
I am wondering if we should link to this website instead. Very similar, but more detailed explanation about each opcode: https://www.evm.codes/#f1

Reply:
ah nice i think this one is better - i will update thks